src/index.js

```diff
@@ -160,9 +167,24 @@ class SSH {
             if (options.onStderr) options.onStderr(chunk)
             output.stderr.push(chunk)
           })
-          if (options.stdin) {
+          if (typeof options.stdin == 'string' && options.stdin) {
             stream.write(options.stdin)
             stream.end()
+          } else if (options.stdin instanceof EventEmitter) {
+            options.stdin.on('data', function(chunk) {
```

Comment:
Instead of all of this, how about just `options.stdin.pipe(stream)`?

Reply:
I was wondering how it would make sense to enable the stream api. This makes sense.